conn.go

```diff
@@ -729,3 +727,20 @@ func (c *Conn) SendKeyUpdate(requestUpdate bool) error {
 func (c *Conn) GetHsState() string {
 	return reflect.TypeOf(c.hState).Name()
 }
+
+func (c *Conn) ComputeExporter(label string, context []byte, keyLength int) ([]byte, error) {
```

Comment:
I just learned something about go today.  Apparently `string` is [essentially the same](https://blog.golang.org/strings) as `[]byte`.  I don't know what to do about that though.